dll/win32/advapi32/advapi32.spec

```diff
@@ -404,12 +407,18 @@
 @ stdcall LsaSetTrustedDomainInfoByName(ptr ptr long ptr)
 @ stdcall LsaSetTrustedDomainInformation(ptr ptr long ptr)
 @ stdcall LsaStorePrivateData(ptr ptr ptr)
-@ stdcall MD4Final(ptr)
-@ stdcall MD4Init(ptr)
-@ stdcall MD4Update(ptr ptr long)
-@ stdcall MD5Final(ptr)
-@ stdcall MD5Init(ptr)
-@ stdcall MD5Update(ptr ptr long)
+@ stdcall -version=0-0x502 MD4Final(ptr)
+@ stdcall -version=0x600+ MD4Final(ptr) NTDLL.MD4Final
+@ stdcall -version=0-0x502 MD4Init(ptr)
+@ stdcall -version=0x600+ MD4Init(ptr) NTDLL.MD4Init
+@ stdcall -version=0-0x502 MD4Update(ptr ptr long)
+@ stdcall -version=0x600+ MD4Update(ptr ptr long) NTDLL.MD4Update
+@ stdcall -version=0-0x502 MD5Final(ptr)
+@ stdcall -version=0x600+ MD5Final(ptr) NTDLL.MD5Final
+@ stdcall -version=0-0x502 MD5Init(ptr)
+@ stdcall -version=0x600+ MD5Init(ptr) NTDLL.MD5Init
+@ stdcall -version=0-0x502 MD5Update(ptr ptr long)
+@ stdcall -version=0x600+ MD5Update(ptr ptr long) NTDLL.MD5Update
```

Comment:
We usually use lowercase for the module name (e.g. `ntdll.MD5Update`)

Reply:
Committed. Note: Windows usually use upper-case for forward thunks, however, it doesn't matter and tooooo nitpick.